internal/pkg/cli/svc_deploy_test.go

```diff
@@ -209,14 +209,31 @@ func TestSvcDeployOpts_Execute(t *testing.T) {
 					},
 				}
 				m.mockEnvFeaturesDescriber.EXPECT().AvailableFeatures().Return([]string{"mockFeature1", "mockFeature2"}, nil)
-				m.mockEnvFeaturesDescriber.EXPECT().Version().Times(0)
+				m.mockEnvFeaturesDescriber.EXPECT().Version().Times(1)
 				m.mockDeployer.EXPECT().UploadArtifacts().Return(&deploy.UploadArtifactsOutput{}, nil)
 				m.mockDeployer.EXPECT().DeployWorkload(gomock.Any()).Return(nil, mockError)
 				m.mockDeployer.EXPECT().IsServiceAvailableInRegion("").Return(false, nil)
 			},
 
 			wantedError: fmt.Errorf("deploy service frontend to environment prod-iad: some error"),
 		},
+		"error if unable to get env version": {
```

Comment:
I'm surprised that we don't have a `"success"` test case like in `svc_package_test.go` 🤔

Reply:
added one! 😊